app/models/job.rb

```diff
@@ -31,4 +35,12 @@ def generate_uuid
   def to_param
     uuid
   end
+
+  def self.ransackable_attributes(auth_object = nil)
+    ["applicable_for", "company_id", "created_at", "description", "id", "job_location", "job_type", "link_to_apply", "posted_by_id", "salary_range", "status", "title", "total_positions", "updated_at", "uuid"]
```

Comment:
can you please define this array as %w() syntax?

Reply:
this seems fine to me? can you please let me know any specific reason so that I need to make such change? As far as I know, both are the fine ways to define array.